### ISSUE TITLE
ML-wrapper for parallel (or successive) training of several ANNs

### ISSUE THREAD

**Issue by mtewes** [megalut/megalut#70]
It would be nice to have a wrapper to train different MLs in parallel. Should be very general of course. Can be used to try out different FANNParams, to perform trainings with the same params (committee), or simply to train with different MLParams (g1, g2, size, flux, ...)
